edge-modules/iotedge-diagnostics-dotnet/src/Program.cs

```diff
@@ -85,52 +73,31 @@ static async Task EdgeAgent(string managementUri)
 
         static async Task Upstream(string hostname, string port, string proxy)
         {
+            var httpClientHandler = new HttpClientHandler();
+            httpClientHandler.ServerCertificateCustomValidationCallback = (message, cert, chain, sslPolicyErrors) => {
+                    return true;   //Is valid
```

Comment:
why do you override this? shouldn't we be checking if the client trusts the proxy? Or is it b/c the diagnostics module doesn't retrieve the trust bundle?

Reply:
Otherwise we need to extract the trusted CA from config.yaml. I did that because it is not the focus of this test and it is faster.